pkg/sql/execinfrapb/processors_ttl.proto

```diff
@@ -40,14 +40,9 @@ message TTLSpec {
     (gogoproto.customname) = "RowLevelTTLDetails"
   ];
 
-  // Deprecated: AOST has been replaced with AOSTDuration.
-  // TODO(ewall): remove in 24.1
```

Comment:
Compatibility was required until 24.1 when this comment was made, but is now only needed until 23.2.